lib/bungee/src/Instrumentation.cpp

```
@@ -0,0 +1,69 @@
+		static const char *names[] = {"specifyGrain", "analyseGrain", "synthesiseGrain"};
+		instrumentation.log("FATAL: stretcher functions called in the wrong order: %s was called when expecting a call to %s", names[sequence], names[instrumentation.expected]);
+		std::abort();
+	}
+	instrumentation.expected = (sequence + 1) % 3;
+
+#ifdef EIGEN_RUNTIME_NO_MALLOC
+	Eigen::internal::set_is_malloc_allowed(false);
+#endif
+}
+
+Instrumentation::Call::~Call()
```

Cppcheck (2.19.0)

[style] The function 'check' is never used.

(unusedFunction)